scripts/tests/web.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
```

Comment:
general comment on this test: if we run this test within earthly (i.e. testing earthly in earthly), then we should do that instead. Typically the outside of earthly tests are only required when the embedded earthly makes use of `WITH DOCKER`.